common/src/main/java/com/wynntils/core/mod/TickSchedulerManager.java

```diff
@@ -43,4 +45,17 @@ public void onTick(TickAlwaysEvent e) {
             }
         }
     }
+
+    @SubscribeEvent
+    public void onTick(TickEvent e) {
+        if (ticks == Integer.MAX_VALUE) {
+            ticks = 0;
+        } else {
+            ticks++;
+        }
+    }
+
+    public int getTicks() {
```

Comment:
I think there is a vanilla function returning the number of ticks since the world start, or whatever. Maybe you can just use that?